### PR TITLE
Making the bundle work with disabled csrf protection

### PR DESCRIPTION
**Pull request by defrag**
So currently if u have application that uses some functional testing, there is a big chance that u 
have csrf protection disabled in your test environment. This PR will fix the issue, where there
is fatal error thrown whenver token manager isnt registered definition.

### CODE REVIEW

File: Features/ensuring_that_disabled_csrf_protection_doesnt_break_the_application.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Feature: Ensuring that disabled csrf protection doesnt break the application
+  In order to test funtionally my application's forms
+  As a developer
+  I need to turn of the csrf protection and things shoulnt break
```
Comment (dunglas) [inline Features/ensuring_that_disabled_csrf_protection_doesnt_break_the_application.feature]: Typo: "off"

**Comment by defrag** [comment]
@dunglas is there anything else to do here?

**Comment by dunglas** [comment]
@defrag not related with this PR but why don't you use https://github.com/Behat/Symfony2Extension?

**Comment by dunglas** [comment]
And can you add some lines in the readme about this PR and the form extension?

**Comment by defrag** [comment]
No real reason, but u dont really need any driver on anything, just handling the request and checking the response, so i didnt bother pinning the sf2 extension. I will open pr with form docs soon, im pretty busy now. This PR is kind of from whats blocking me of replacing my own system for handling csrf with yours. I would like to do it soon :)

**Comment by dunglas** [comment]
thanks @defrag. I'll tag a new release when doc will be added.

**Pull request merged**